### DISCUSSION TITLE
Is there a way to get something like magit-list-repo?

### DISCUSSION THREAD

**Discussion by stefanocoretta** [NeogitOrg/neogit#1238] [Q&A] [answered]
Hi! I wonder if there's a way of getting a buffer like magit-list-repo, which a list of repos and status icons, from which you can jump to the repo directory?

https://magit.vc/manual/magit/Repository-List.html

**Comment by CKolkey** [comment, marked as answer]
Ah, that's a new one to me. Hmm. No, there's not a way to do that currently, and I don't think I'm super keen to add session management to the plugin since there are plenty of good existing plugins that manage projects/sessions in neovim... and I don't even use those.

**Reply by stefanocoretta** [reply]
Thanks! Do you have pointers as to which other plugins might achieve that? (I'm new to neovim, I'm seeing whether it would work for me so I'm just dipping my toes for the moment!)

**Reply by CKolkey** [reply]
I don't have first hand experience with them, but you can find some options here:
https://github.com/rockerBOO/awesome-neovim?tab=readme-ov-file#project

I'd look for one with a telescope integration (and check out telescope.nvim if you haven't already) :)
